deepchecks/core/suite.py

```diff
@@ -291,31 +295,59 @@ def to_wandb(
         with wandb_run(**wandb_kwargs) as run:
             run.log(WandbSerializer(self).serialize())
 
-    def get_checks_not_ran(self) -> List[CheckFailure]:
+    def get_not_ran_checks(self) -> List[CheckFailure]:
         """Get all the check results which did not run (unable to run due to missing parameters, exception, etc).
 
         Returns
         -------
         List[CheckFailure]
             All the check failures in the suite.
         """
-        return self.select_results(self.failures)
+        return cast(List[CheckFailure], self.select_results(self.failures))
```

Comment:
why cast?

Reply:
to prevent the static type checker from complaining

`select_results` returns value of type `List[Union[CheckResult, CheckFailure]]`
which is incompetiable with return result of that function `List[CheckFailure]`